app/components/license-form-toggle-button/template.hbs

```diff
@@ -1 +1 @@
-<button class="btn btn-default {{if isSelected "active"}}" type="button" {{action "click"}}>{{text}}</button>
+<button class="btn btn-default {{text}} {{if isSelected "active"}}" type="button" {{action "click"}}>{{text}}</button>
```

Comment:
I am sharing text value as class name, so this button is going to have class `"btn btn-default yes active"` (or no)  maybe it would be better to pass the parameter for class name?